packages/calculator/__mocks__/@pie-framework/material-ui-calculator.jsx

```diff
@@ -0,0 +1,5 @@
+const indicators = {
```

Comment:
there is no `indicators` in `@pie-framework/material-ui-calculator` ? this looks like it is a copy of a mock for `@pie-lib/icons`?